cargo-public-api/Cargo.toml

```
@@ -11,6 +11,12 @@ categories = ["command-line-utilities", "development-tools::cargo-plugins"]
 license = "MIT"
 repository = "https://github.com/Enselic/cargo-public-api"
 
+[features]
+default = ["diff-latest"]
+# Enable `cargo public-api diff latest`. Requires 33 more deps (most notably the
+# `git2` crate) so that the crates.io index can be updated and read.
+diff-latest = ["crates-index"]
```

```suggestion
diff-latest = ["dep:crates-index"]
```

this avoids having a feature `diff-latest` and `crates-index`

---

Thanks, I didn't know about this actually :blush:

---

`dep:` was introduced in 1.60, see https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies for more (where more is being able to do `my-feature = ["package-name?/feature-name"]` to only enable `package-name/feature-name` if `package-name` is also enabled)